### PR TITLE
[ContactDetails] Relegate debug log to verbose output tier

### PR DESCRIPTION
**Pull request by brianblakely**
If this log is necessary for debugging, it should use the `debug` method, which will only be visible when "Verbose" messages are enabled in the dev console.

### CODE REVIEW

**Comment by joshwooding** [comment]
This ContactDetails component will be removed in a future release and has been replaced by a pattern: https://www.saltdesignsystem.com/salt/patterns/contact-details

In this case we can delete the log as it doesn't need to be there. But in order for me to accept this PR you'll have to go through some pre-requisites. I'll contact you personally to walk you through them.

**Review by origami-z — APPROVED** [review]

**Comment by origami-z** [comment]
@brianblakely, contact detail component in lab will be removed and replaced with [the pattern documented](https://www.saltdesignsystem.com/salt/patterns/contact-details). i.e. you'll assemble your own

File: packages/lab/src/contact-details/ContactDetails.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ export const ContactDetails = forwardRef<HTMLDivElement, ContactDetailsProps>(
 
     variant === "compact" &&
       isStacked &&
-      console.log("componentSize", componentSize.width, containerRef.current);
+      console.debug("componentSize", componentSize.width, containerRef.current);
```
Comment (joshwooding) [inline packages/lab/src/contact-details/ContactDetails.tsx:60]: As we discussed in our chat, we can remove this completely :)

File: .changeset/real-seas-end.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/lab": patch
+---
+
+[ContactDetails] Relegate debug log to verbose output tier.
```
Comment (joshwooding) [inline .changeset/real-seas-end.md:5]:
```suggestion
Removed a `console.log` call from `ContactDetails`.
```

**Comment by brianblakely** [comment]
Per above comments, because this component will not be carrying on, I am closing this PR.

**Pull request closed without merging**